.github/CODEOWNERS

```diff
@@ -1,5 +1,5 @@
-# Cast
-/crates/cast/               @foundry-rs/starknet-foundry-cast
+# chain-interactions
+/crates/sncast/             @foundry-rs/starknet-foundry-cast
```

Comment:
The team could be renamed to `chain-interactions` as well, but that's for a separate PR.